problem_3/lib/largest_prime_factor.rb

```diff
@@ -0,0 +1,24 @@
+#What is the largest prime factor of the number 600851475143 ?
+
+class PrimeFactor
+
+	def initialize(limit)
+		@limit = limit
+	end
+
+	def factor_finder
+		div = 2
+		largest_factor = 0
+		while (div <= @limit)
+			if (@limit % div == 0)
+				@limit = @limit / div
+				largest_factor = div
+				div += 1
+			end
+			div += 1
+		end
+		largest_factor
```

Comment:
Does this return the largest prime factor? It seems like this function is only returning the largest factor

Reply:
I was checking the tests against an online calculator so it should be returning the largest prime factor (math is definitely something I'm working on so I could be wrong)